gnuradio-runtime/lib/qa_circular_file.cc

```diff
@@ -24,9 +24,8 @@
 #include "config.h"
 #endif
 
-#include "qa_circular_file.h"
 #include "circular_file.h"
-#include <cppunit/TestAssert.h>
+#include <boost/test/unit_test.hpp>
 #include <iostream>
 #include <stdio.h>
 #include <unistd.h>
```

Comment:
we use that for `unlink`. We shouldn't be using that in the first place¹, but for now, keep.

¹: `#include <cstdlib>; std::tmpfile` would be **far** better (no race conditions on parallel tests), but I'll do a separate issue on that.)